src/main/features/dashboard/views/macro/claimStatus/helpWithFees.njk

```diff
@@ -223,7 +223,7 @@
 </div>
 {% endmacro %}
 
-{% macro helpWithFeesIntrestRecalculateStatus(claim) %}
+{# {% macro helpWithFeesIntrestRecalculateStatus(claim) %}
```

Comment:
If this code is no longer required (removed as part of your story), please remove it instead of commenting it.